packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -247,6 +268,13 @@ type ProjectNavProps = Pick<
   | 'environment'
   | 'alertPollingInterval'
 > & {
+  admin?: boolean;
+  activeProduct?: Product;
+  activeNav?: ActiveNavElement;
+  onProjectChange?: (onChangeInterface: OnChangeInterface) => void;
+  mode?: Mode;
+  alertPollingInterval?: number;
+  environment?: Environment;
```

Comment:
curious about why this change is needed. does the `Pick` above not already include these for us?

Reply:
I thought the same thing but `yarn fix` started throwing errors about those props not being defined.

Reply:
ah, seems like a linter false-positive. you can wrap those lines in
```
/* eslint-disable react/prop-types */
...
/* eslint-enable react/prop-types */
```